modules/scip/9.2.0.bcr.3/presubmit.yml

```
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
```

```suggestion
  - 8.x
```

6 doesn't support overlays

---

Done.